api/models/ingredientsModel.js

```diff
@@ -0,0 +1,37 @@
+const db = require('../../data/dbConfig.js')
+
+module.exports = {
+    find,
+    findById,
+    add,
+    update,
+    remove
+}
+
+function find() {
+    return db('ingredients')
+}
+
+function findById(id) {
+    return db('ingredients')
+        .where({ id })
+        .first()
+}
+
+async function add(ingredient) {
+    const [id] = await db('ingredients').insert(ingredient)
+
+    return findById(id)
+}
+
+function update(id, changes) {
+    return db('ingredients')
+        .where({ id })
+        .update(changes, '*')

```

Comment:
Same thing here, no need for *, and you might want to return the ingredient by id back